calculate_baseline_metrics.py

```diff
@@ -122,31 +122,42 @@ def parse_args():
     parser.add_argument('--label_file', type=str,
-                        help='Path to hdf5/bw file containing labels')
+                        help='Path to hdf5/bigWig file containing labels',
+                        required=True)
     parser.add_argument('--test_file', type=str,
-                        help='Path to hdf5/bw file containing labels. Assumed \
-                                to be present in label_file by default.')
-    parser.add_argument('--peak_file', type=str,
-                        help='Path to hdf5/bw file with peak labels. Assumed \
+                        help='Path to hdf5/bigWig file containing labels. Assumed \
                                 to be present in label_file by default.')
     parser.add_argument('--task', type=str,
                         choices=('regression', 'classification'),
-                        help='determines metrics')
+                        help='determines the metrics to be calculated')
     parser.add_argument('--ratio', type=float, help='subsampling ratio')
     parser.add_argument('--sep_peaks', action='store_true',
-                        help='separate regression metrics for\
-                                peaks and non-peaks')
+                        help='calculate separate regression metrics for \
+                                peaks and non-peaks.')
+    parser.add_argument('--peak_file', type=str,
+                        help='Path to hdf5/bigWig file with peak labels. Assumed \
+                                to be present in label_file by default. \
+                                Use only with --sep_peaks.')
     parser.add_argument('--thresholds', type=str,
-                        help='threshold or list of thresholds for\
-                                classification metrics')
+                        help='threshold or list of thresholds for \
+                                classification metrics. If supplying a list, \
+                                use the form: "[threshold1, threshold2,...]"')
     parser.add_argument('--auc', action='store_true',
-                        help='calculate AUC metrics')
+                        help='calculate AUROC and AUPRC metrics')
     parser.add_argument('--intervals', type=str,
-                        help='Intervals to read bigWig files')
+                        help='Path to BED file containing genomic intervals. \
+                        Data for these intervals will be read from all \
+                        bigWig files supplied.')
     parser.add_argument('--sizes', type=str,
-                        help='Chromosome sizes to read bigWig file')
+                        help='Path to chromosome sizes file. \
+                        If supplied, data for all chromosomes in \
+                        the sizes file will be read from all bigWig files. \
+                        Only used if --intervals is not supplied.')
```

Comment:
from the description, I'm not clear about the difference between --sizes and --intervals. Could you explain further (as a reply to this comment, for my understanding)

Reply:
This script will calculate metrics over a specified part of the genome, e.g. positions 1-100,000 on chromosome 1. This will be specified as intervals (--intervals). 
Alternatively, you may want to calculate metrics over the whole length of one or more chromosomes. In this case you supply the list of chromosomes as a sizes file, and the script calculates the metrics over the complete length of those chromosomes.

Reply:
Oh, that's very helpful, thanks! could you incorporate this into the help of sizes and intervals, so it's more intuitive.

Reply:
Done